src/server.cpp

```diff
@@ -87,8 +88,17 @@ Packet Server::makeResponse(const Packet& request)
     std::cout << "Request packet\n";
     printPacket(request);
 
+    std::vector<Attribute*> attributes;
+    attributes.push_back(new String(USER_NAME, "test"));
+    attributes.push_back(new Integer(NAS_PORT, 20));
+    std::array<uint8_t, 4> address {127, 104, 22, 17};
+    attributes.push_back(new IpAddress(NAS_IP_ADDRESS, address));
+    attributes.push_back(new Encrypted(USER_PASSWORD, "password123"));
+    std::vector<uint8_t> bytes {'1', '2', '3', 'a', 'b', 'c'};
```

Comment:
Let's add some non-printable data:
```c++
std::vector<uint8_t> bytes{0, 13, 42, 'a', 'b', 'c'};
```
Just to make sure we always treat this type as abstract data, not anything printable.